includes/class-woocommerce.php

```diff
@@ -227,6 +227,29 @@ private function define( $name, $value ) {
 		}
 	}
 
+	/**
+	 * Returns true if the request is a non-legacy REST API request.
+	 *
+	 * Legacy REST requests should run extra code for backwards compatibility.
+	 *
+	 * @return bool
+	 */
+	private static function is_nonlegacy_rest_api_request() {
```

Comment:
We can go with just `is_rest_api_request()`, no need to mention about our legacy REST API since we should remove it anyway.

Reply:
Sure, updated.